Cython/Includes/cpython/datetime.pxd

```diff
@@ -119,21 +228,39 @@ cdef extern from "datetime.h":
     int PyDateTime_DATE_GET_MINUTE(object o)
     int PyDateTime_DATE_GET_SECOND(object o)
     int PyDateTime_DATE_GET_MICROSECOND(object o)
+    int PyDateTime_DATE_GET_FOLD(object o)
+    object PyDateTime_DATE_GET_TZINFO(object o)
 
     # Getters for time (C macros).
     int PyDateTime_TIME_GET_HOUR(object o)
     int PyDateTime_TIME_GET_MINUTE(object o)
     int PyDateTime_TIME_GET_SECOND(object o)
     int PyDateTime_TIME_GET_MICROSECOND(object o)
+    int PyDateTime_TIME_GET_FOLD(object o)
+    object PyDateTime_TIME_GET_TZINFO(object o)
 
     # Getters for timedelta (C macros).
     int PyDateTime_DELTA_GET_DAYS(object o)
     int PyDateTime_DELTA_GET_SECONDS(object o)
     int PyDateTime_DELTA_GET_MICROSECONDS(object o)
 
+    # Constructors
+    object PyTimeZone_FromOffset(object offset)
+    object PyTimeZone_FromOffsetAndName(object offset, str name)
```

Comment:
We shouldn't require users to pass a `str` here _and_ require Cython to know or test it.
```suggestion
    object PyTimeZone_FromOffsetAndName(object offset, object name)
```